### ISSUE TITLE
【EESS】New submissions for Wed, 27 Sep 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#686]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
### Applications of Sequential Learning for Medical Image Classification
 - **Authors:** Sohaib Naim, Brian Caffo, Haris I Sair, Craig K Jones
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2309.14591
 - **Pdf link:** https://arxiv.org/pdf/2309.14591
 - **Abstract**
 Purpose: The aim of this work is to develop a neural network training framework for continual training of small amounts of medical imaging data and create heuristics to assess training in the absence of a hold-out validation or test set. Materials and Methods: We formulated a retrospective sequential learning approach that would train and consistently update a model on mini-batches of medical images over time. We address problems that impede sequential learning such as overfitting, catastrophic forgetting, and concept drift through PyTorch convolutional neural networks (CNN) and publicly available Medical MNIST and NIH Chest X-Ray imaging datasets. We begin by comparing two methods for a sequentially trained CNN with and without base pre-training. We then transition to two methods of unique training and validation data recruitment to estimate full information extraction without overfitting. Lastly, we consider an example of real-life data that shows how our approach would see mainstream research implementation. Results: For the first experiment, both approaches successfully reach a ~95% accuracy threshold, although the short pre-training step enables sequential accuracy to plateau in fewer steps. The second experiment comparing two methods showed better performance with the second method which crosses the ~90% accuracy threshold much sooner. The final experiment showed a slight advantage with a pre-training step that allows the CNN to cross ~60% threshold much sooner than without pre-training. Conclusion: We have displayed sequential learning as a serviceable multi-classification technique statistically comparable to traditional CNNs that can acquire data in small increments feasible for clinically realistic scenarios.
### Challenges of building medical image datasets for development of deep  learning software in stroke
 - **Authors:** Alessandro Fontanella, Wenwen Li, Grant Mair, Antreas Antoniou, Eleanor Platt, Chloe Martin, Paul Armitage, Emanuele Trucco, Joanna Wardlaw, Amos Storkey
 - **Subjects:** Image and Video Processing (eess.IV)
 - **Arxiv link:** https://arxiv.org/abs/2309.15081
 - **Pdf link:** https://arxiv.org/pdf/2309.15081
 - **Abstract**
 Despite the large amount of brain CT data generated in clinical practice, the availability of CT datasets for deep learning (DL) research is currently limited. Furthermore, the data can be insufficiently or improperly prepared for machine learning and thus lead to spurious and irreproducible analyses. This lack of access to comprehensive and diverse datasets poses a significant challenge for the development of DL algorithms. In this work, we propose a complete semi-automatic pipeline to address the challenges of preparing a clinical brain CT dataset for DL analysis and describe the process of standardising this heterogeneous dataset. Challenges include handling image sets with different orientations (axial, sagittal, coronal), different image types (to view soft tissues or bones) and dimensions, and removing redundant background. The final pipeline was able to process 5,868/10,659 (45%) CT image datasets. Reasons for rejection include non-axial data (n=1,920), bone reformats (n=687), separated skull base/vault images (n=1,226), and registration failures (n=465). Further format adjustments, including image cropping, resizing and scaling are also needed for DL processing. Of the axial scans that were not localisers, bone reformats or split brains, 5,868/6,333 (93%) were accepted, while the remaining 465 failed the registration process. Appropriate preparation of medical imaging datasets for DL is a costly and time-intensive process.
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
## Keyword: rendering
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
### Simultaneously Learning Speaker's Direction and Head Orientation from  Binaural Recordings
 - **Authors:** Harshvardhan Takawale, Nirupam Roy
 - **Subjects:** Audio and Speech Processing (eess.AS); Sound (cs.SD); Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2309.15064
 - **Pdf link:** https://arxiv.org/pdf/2309.15064
 - **Abstract**
 Estimation of a speaker's direction and head orientation with binaural recordings can be a critical piece of information in many real-world applications with emerging `earable' devices, including smart headphones and AR/VR headsets. However, it requires predicting the mutual head orientations of both the speaker and the listener, which is challenging in practice. This paper presents a system for jointly predicting speaker-listener head orientations by leveraging inherent human voice directivity and listener's head-related transfer function (HRTF) as perceived by the ear-mounted microphones on the listener. We propose a convolution neural network model that, given binaural speech recording, can predict the orientation of both speaker and listener with respect to the line joining the two. The system builds on the core observation that the recordings from the left and right ears are differentially affected by the voice directivity as well as the HRTF. We also incorporate the fact that voice is more directional at higher frequencies compared to lower frequencies.
## Keyword: retrieval
There is no result 
## Keyword: video retrieval
There is no result 
### Learning-Based Latency-Constrained Fronthaul Compression Optimization in  C-RAN
 - **Authors:** Axel Grönland, Alessio Russo, Yassir Jedra, Bleron Klaiqi, Xavier Gelabert
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2309.15060
 - **Pdf link:** https://arxiv.org/pdf/2309.15060
 - **Abstract**
 The evolution of wireless mobile networks towards cloudification, where Radio Access Network (RAN) functions can be hosted at either a central or distributed locations, offers many benefits like low cost deployment, higher capacity, and improved hardware utilization. The flexibility in the functional deployment comes at the cost of stringent fronthaul (FH) capacity and latency requirements. One possible approach to deal with these rigorous constraints is to use FH compression techniques. Due to varying FH load, FH compression needs to be adaptive, i.e., apply more FH compression in high load to meet FH capacity and latency requirements and less compression in medium and low load to improve FH utilization and throughput performance. In this paper, a model-free deep reinforcement learning (DRL) based FH compression (DRL-FC) framework is proposed that dynamically controls FH compression through various configuration parameters such as modulation order, precoder granularity, and precoder weight quantization that affect both FH load and throughput performance. Simulation results show that DRL-FC exhibits significantly higher FH utilization (70.2% on average) and throughput than a reference scheme across different FH load levels. At the same time, the proposed DRL-FC framework is able to meet the predefined FH latency constraints under various FH loads.
## Keyword: smartphone
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### Unveiling Fairness Biases in Deep Learning-Based Brain MRI  Reconstruction
 - **Authors:** Yuning Du, Yuyang Xue, Rohan Dharmakumar, Sotirios A. Tsaftaris
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2309.14392
 - **Pdf link:** https://arxiv.org/pdf/2309.14392
 - **Abstract**
 Deep learning (DL) reconstruction particularly of MRI has led to improvements in image fidelity and reduction of acquisition time. In neuroimaging, DL methods can reconstruct high-quality images from undersampled data. However, it is essential to consider fairness in DL algorithms, particularly in terms of demographic characteristics. This study presents the first fairness analysis in a DL-based brain MRI reconstruction model. The model utilises the U-Net architecture for image reconstruction and explores the presence and sources of unfairness by implementing baseline Empirical Risk Minimisation (ERM) and rebalancing strategies. Model performance is evaluated using image reconstruction metrics. Our findings reveal statistically significant performance biases between the gender and age subgroups. Surprisingly, data imbalance and training discrimination are not the main sources of bias. This analysis provides insights of fairness in DL-based image reconstruction and aims to improve equity in medical AI applications.
### Effective Multi-Agent Deep Reinforcement Learning Control with Relative  Entropy Regularization
 - **Authors:** Chenyang Miao, Yunduan Cui, Huiyun Li, Xinyu Wu
 - **Subjects:** Systems and Control (eess.SY); Artificial Intelligence (cs.AI); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2309.14727
 - **Pdf link:** https://arxiv.org/pdf/2309.14727
 - **Abstract**
 In this paper, a novel Multi-agent Reinforcement Learning (MARL) approach, Multi-Agent Continuous Dynamic Policy Gradient (MACDPP) was proposed to tackle the issues of limited capability and sample efficiency in various scenarios controlled by multiple agents. It alleviates the inconsistency of multiple agents' policy updates by introducing the relative entropy regularization to the Centralized Training with Decentralized Execution (CTDE) framework with the Actor-Critic (AC) structure. Evaluated by multi-agent cooperation and competition tasks and traditional control tasks including OpenAI benchmarks and robot arm manipulation, MACDPP demonstrates significant superiority in learning capability and sample efficiency compared with both related multi-agent and widely implemented signal-agent baselines and therefore expands the potential of MARL in effectively learning challenging control scenarios.
### Exploring the impact of automated vehicles lane-changing behavior on  urban network efficiency
 - **Authors:** Alberto Pelizza, Federico Orsini, Sefa Yilmaz-Niewerth, Riccardo Rossi, Bernhard Friedrich
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2309.14731
 - **Pdf link:** https://arxiv.org/pdf/2309.14731
 - **Abstract**
 While automated vehicle (AV) research has grown steadily in recent years, the impact of automated lane changing behavior on transportation systems remains a largely understudied topic. The present work aims to explore the effects of automated lane changing behavior on urban network efficiency as the penetration rate of AVs increases. To the best of the authors knowledge, this represents the first attempt to do so by isolating the effects of the lane changing behavior; this was obtained by considering AVs with automated lateral control, yet retaining the same longitudinal control characteristics of conventional vehicles (CV). An urban road network located in Hannover, Germany, was modeled with the microsimulation software SUMO, and several scenarios were analyzed, starting from a baseline with only CVs and then progressively increasing the AV penetration rate with 10% increments. Results highlight a modest, but statistically significant, decrease in system performance, with travel times increasing, and average speed and network capacity decreasing, as penetration rates increase. This was likely caused by a more prudent behavior of AVs, which accepted larger gaps than CVs when performing lane changing maneuvers.
### Learning-Based Latency-Constrained Fronthaul Compression Optimization in  C-RAN
 - **Authors:** Axel Grönland, Alessio Russo, Yassir Jedra, Bleron Klaiqi, Xavier Gelabert
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2309.15060
 - **Pdf link:** https://arxiv.org/pdf/2309.15060
 - **Abstract**
 The evolution of wireless mobile networks towards cloudification, where Radio Access Network (RAN) functions can be hosted at either a central or distributed locations, offers many benefits like low cost deployment, higher capacity, and improved hardware utilization. The flexibility in the functional deployment comes at the cost of stringent fronthaul (FH) capacity and latency requirements. One possible approach to deal with these rigorous constraints is to use FH compression techniques. Due to varying FH load, FH compression needs to be adaptive, i.e., apply more FH compression in high load to meet FH capacity and latency requirements and less compression in medium and low load to improve FH utilization and throughput performance. In this paper, a model-free deep reinforcement learning (DRL) based FH compression (DRL-FC) framework is proposed that dynamically controls FH compression through various configuration parameters such as modulation order, precoder granularity, and precoder weight quantization that affect both FH load and throughput performance. Simulation results show that DRL-FC exhibits significantly higher FH utilization (70.2% on average) and throughput than a reference scheme across different FH load levels. At the same time, the proposed DRL-FC framework is able to meet the predefined FH latency constraints under various FH loads.
